backend/LexBoxApi/Auth/Requirements/AccessProjectUsersRequirementHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿using HotChocolate.Resolvers;
+using LexCore.Entities;
+using LexCore.ServiceInterfaces;
+using Microsoft.AspNetCore.Authorization;
+
+namespace LexBoxApi.Auth.Requirements;
+
+public class AccessProjectUsersRequirement : IAuthorizationRequirement;
+
+public class AccessProjectUsersRequirementHandler(
+    IPermissionService permissions,
+    ILogger<AccessProjectUsersRequirementHandler> logger
+) : AuthorizationHandler<AccessProjectUsersRequirement>
+{
+    protected override async Task HandleRequirementAsync(AuthorizationHandlerContext context,
+        AccessProjectUsersRequirement requirement)
+    {
+        //todo get guid.
```

Comment:
Is this todo comment completed by the `if (context.Resource is IMiddlewareContext middlewareContext)` block below? Or is this still a todo?

Reply:
ah yes, that is invalid now, thanks for catching that.